### DISCUSSION TITLE
NeMo raises LightningNotInstalledException

### DISCUSSION THREAD

**Discussion by SeniorGlassMaster** [Lightning-AI/pytorch-lightning#10812] [code help: NLP / ASR / TTS]
actually, pytorch_lightning, hydra, and omegaconf are already installed, and _import _nemo.collections.asr as nemo_asr__  is no issues, however, while run _vad_model = nemo_asr.models.EncDecClassificationModel(cfg=config.model, trainer=trainer)_ failed. No similar bug can be found.
it is weird, i can not understand.
has anyone konw how to solve it? 

![image](https://user-images.githubusercontent.com/30117356/143863050-1d63281b-caec-41ea-905a-46d735c82fd8.png)

**Comment by rohitgr7** [comment]
the issue is coming from the NeMo package. Mind raising an issue there?
https://github.com/NVIDIA/NeMo
